src/chameleon/compiler.py

```diff
@@ -405,7 +405,7 @@ def __call__(self, name, engine):
                 target = template(
                     "translate(msgid, mapping=mapping, domain=__i18n_domain, context=__i18n_context, target_language=target_language)",
                     msgid=ast.Str(s=formatting_string),
-                    target_language=target_language,
+                    target_language="target_language",
```

Comment:
I think this should be:
```diff
-                    target_language="target_language",
+                    target_language=load("target_language"),
```

Reply:
Both work, but I guess then it should also be used in the ``if len(nodes) == 1:`` case a few lines above.